README.md

```diff
@@ -1788,6 +1789,43 @@ Map::from( ['a' => 'X', 'b' => 'Y'] )->has( 'X' );
 ```
 
 
+### if()
+
+Executes callables $then and $else depending on callable $condition.
+
+```php
+public function if( Callable $condition, Callable $then, $Callable $else = null ) : bool
```

Comment:
The type hint for the first parameter must be removed too

Reply:
fixed in c329afdda114015b900470c1ef73edc6439fc1a1